libUnderworld/PICellerator/MaterialPoints/src/GeneralSwarm.cpp

```diff
@@ -434,3 +442,110 @@ void GeneralSwarm_ClearSwarmMaps( void* swarm ) {
     }
 }
 
+struct GeneralSwarm_nanoflann_interface
+{
+    GeneralSwarm* swarm;
```

Comment:
@jmansour I don't see the need for indents. If required consider use { ... } to wrap it.

Reply:
@julesghub, yep ew.  

invisibles on!  whitespace fixed!

Reply:
Wonderful!
Perhaps a gist for the working example, they can be made private i think.